src/bokeh/models/annotations/geometry.py

```diff
@@ -111,6 +113,44 @@ def __init__(self, *args, **kwargs) -> None:
     default.
     """)
 
+    border_radius = BorderRadius(default=0, help="""
+    Allows the box to have rounded corners.
+
+    .. note::
+        This property is experimental and may change at any point.
+    """)
```

Comment:
For my own future reference, do you prefer

```python
border_radius = Experimental(BorderRadius(...))
```
or
```python
border_radius = BorderRadius(..., experimental=True)
```
or 
```python
border_radius = BorderRadius(...).experimental()
```
or some other kind of programmatic marking? 

I don't really like the first, since "experimental" is not a type, per se.

Reply:
Definitely not the second one, as it "infects" APIs, and I've been working on removing this pattern from bokeh. Probably the first, which is similar to `Alias` and `DeprecatedAlias` and some other. If I was to implement this, I would choose the first one.